lib/save.js

```diff
@@ -37,8 +38,13 @@ function onSave(ops) {
   }
 
   // Pass saves through the test for the pagelist
-  if (pageList.filterForPageOps(ops).length) {
-    pageList.updatePageList(ops);
+  if (pageList.filterForPageOps(_.cloneDeep(ops)).length) {
+    pageList.updatePageList(_.cloneDeep(ops));
```

Comment:
I cloned twice just to make sure I wasn't mutating anything more as a test of a bug I encountered. I'm going to comment it if it truly is a bug, but I wanted to get something out for Nelson. Since the tests have to filter down a whole array there's a possibility that the ops object is getting filtered down before its sent to the handlers for indexing logic, which I need to prevent. The thing I'll probably do is just clone once for the pagelist test and once for the userlist test and then send the original ops payload to the handlers.